### ISSUE TITLE
Sensors show as unavailable

### ISSUE THREAD

**Issue by robbavel** [Chouffy/home_assistant_tgtg#18]
is it correct that the 'store sensors' show as unavailable when no TGTG box is available?

![image](https://user-images.githubusercontent.com/86732548/150651990-63d2ae31-5250-4b25-b618-11979cdb73b2.png)

**Comment by Chouffy** [comment]
No, it means that Home Assistant is unable to get a value. Reasons can be multiple, have you checked home assistant logs?

**Comment by robbavel** [comment]
Fixed it by adding the item ID's manually instead of letting it grab my favorites

**Comment by Chouffy** [comment]
Awesome! I'll close it, and I've updated the readme

**Comment by Benrens** [comment]
I'm experiencing the same thing, but even after I added items manually i'm still getting Unavailable.
![image](https://user-images.githubusercontent.com/7370880/153208331-f00353de-0c3c-4f7c-b5e0-128028ff8492.png)

I'm running HA in docker.

the only log:

2022-02-09 14:08:58 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration tgtg which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
